actionview/lib/action_view/template/handlers.rb

```diff
@@ -44,7 +44,7 @@ def register_template_handler(*extensions, handler)
         end
 
         unless params.find_all { |type, _| type == :req || type == :opt }.length >= 2
-          ActiveSupport::Deprecation.warn <<~eowarn
+          ActiveSupport::Deprecation.warn <<~eowarn.squish
           Single arity template handlers are deprecated.  Template handlers must
           now accept two parameters, the view object and the source for the view object.
           Change:
```

Comment:
This message expects as multi lines, should not be added `squish`.

Reply:
Thanks :smile: Fixed it.